src/net/sf/freecol/client/gui/CanvasMouseListener.java

```diff
@@ -78,7 +78,14 @@ public void mousePressed(MouseEvent e) {
         }
 
         switch (e.getButton()) {
-        case MouseEvent.BUTTON1: // Drag and selection
+        case MouseEvent.BUTTON1: 
+            // If we have GoTo mode enabled then GoTo takes precedence
+            if (getGUI().isGotoStarted()) {
```

Comment:
The method only returns false? So this would never evaluate to true, or am I missing something?

Reply:
The base class returns false, however the subclass SwingGUI checks the canvas if Goto is enabled and returns true. SwingGUI is the object type that is evaluated at runtime when it performs that check in the mouse event